src/flows/parser_item_html_text_streaming_flow.py

```diff
@@ -41,16 +43,22 @@
 on_error=ABORT_STATEMENT;
 """
 
+# get flow db and schema names from config for ordering tasks
 DATABASE_NAME = ARTICLES_DB_SCHEMA_DICT[ENVIRONMENT]["articles_database_name"]
 SCHEMA_NAME = ARTICLES_DB_SCHEMA_DICT[ENVIRONMENT]["articles_schema_name"]
 
+# create copy of default dict and add flow db and schema names for ordering tasks
+FLOW_SNOWFLAKE_DICT = deepcopy(SNOWFLAKE_DEFAULT_DICT)
+FLOW_SNOWFLAKE_DICT["database"] = DATABASE_NAME
+FLOW_SNOWFLAKE_DICT["schema"] = SCHEMA_NAME
+
+
 # SQL statements to insert new ordered raw data
 # These depend on the initial execution of the 'initial clean' statements in the sql folder
 # This will execute after the copy statement as a single statement
-
-INSERT_ORDERED_SQL = f"""set max_date = (select max(snowflake_loaded_at) from {DATABASE_NAME}.{SCHEMA_NAME}.article_content_ordered_live);
+INSERT_ORDERED_SQL = f"""set max_date = (select max(snowflake_loaded_at) from article_content_ordered_live);
```

Comment:
Nit: I don't think this needs to be an f-string anymore.
```suggestion
INSERT_ORDERED_SQL = """set max_date = (select max(snowflake_loaded_at) from article_content_ordered_live);
```

Reply:
ah I will fix this

Reply:
this is fixed